lib/ansible/modules/hostname.py

```diff
@@ -459,42 +403,14 @@ def set_permanent_hostname(self, name):
             f.close()
 
 
-class OpenBSDStrategy(GenericStrategy):
+class OpenBSDStrategy(DebianStrategy):
```

Comment:
considering you are applying it to BSD .. i would move the code from Debian strategy to something like posix|gnu|oldunixy|presystemd/strategy and reasing to all those with the common approach

Reply:
HostnameCLIStrategy?